InitGui.py

```diff
@@ -50,6 +50,7 @@ class SMWorkbench(Workbench):
     global smWBpath
     global smWB_icons_path
 
+    # TODO: "SheetMetal" or "Sheet Metal"

```

Comment:
Should the WB name be join or separated?

Reply:
I have no preference....

Reply:
I'll leave as is